docs/src/pages/components/authenticator/overrides/username.angular.mdx

```diff
@@ -1,11 +1,50 @@
 _app.component.html_
 
-```html{1} file=../../../../../../examples/angular/src/pages/ui/components/authenticator/sign-up-with-email/sign-up-with-email.component.html
+```html{1}
+<amplify-authenticator
+  [services]="services"
+  initialState="signUp"
+>
+  <ng-template
+    amplifySlot="authenticated"
+    let-user="user"
+    let-signOut="signOut"
+  >
+    <h2>Welcome, {{ user.username }}!</h2>
+    <button (click)="signOut()">Sign Out</button>
+  </ng-template>
+</amplify-authenticator>
```

Comment:
I updated the sample code here because the `sign-up-with-email` example showed a lot of irrelevant information for this example.